code/Test_definitions/geofencing-subscriptions.feature

```diff
@@ -13,370 +12,500 @@ Feature: Camara Geofencing Subscriptions API, vwip - Operations on subscriptions
-    Given the resource "{apiroot}/geofencing-subscriptions/vwip/" as geofencing base-url
+    Given the resource "{apiroot}/geofencing-subscriptions/vwip/subscriptions" as geofencing base-url
     And the header "Authorization" is set to a valid access token
     And the header "x-correlator" complies with the schema at "#/components/schemas/XCorrelator"
 
-############################ Happy Path Scenarios ########################
-  @geofencing_subscriptions_01_Create_geofencing_subscription_for_a_device_sync
-  Scenario:  Create geofencing subscription (sync creation)
-    Given that subscriptions are created synchronously
-    And a valid subscription request body 
+  # Success scenarios
+
+  @geofencing_subscriptions_01.1_sync_creation_2legs
+  Scenario Outline: Create geofencing subscription (async creation) with 2-legged-token
+    Given the header "Authorization" is set to a valid access token which does not identify any device
+    And the request body is compliant with the OAS schema at "#/component/schemas/SubscriptionRequest"
     When the request "createGeofencingSubscription" is sent
+    And request property "$.types" is one of the allowed values "<subscription-creation-types>"
+    And request property "$.protocol" is equal to "HTTP"
+    And a valid phone number identified by "$.config.subscriptionDetail.device.phoneNumber"
+    And request property "$.sink" is set to a valid callbackUrl
     Then the response code is 201
-    And the response header "Content-Type" is "application/json"
-    And the response header "x-correlator" has same value as the request header "x-correlator"
-    And the response body complies with the OAS schema at "/components/schemas/Subscription"
+    And the response body complies with the OAS schema at "#/components/schemas/Subscription"
+    And the response properties "$.types", "$.protocol", "$.sink" and "$.config.subscriptionDetail.device.phoneNumber" are present with the values provided in the request
+    And the response property "$.id" is present
+    And the response property "$.startsAt" and "$.expiresAt", if present, has a valid value with date-time format
+    And the response property "$.status", if present, has the value "ACTIVATION_REQUESTED", "ACTIVE" or "INACTIVE"
+
+    Examples:
+      | subscription-creation-types                                |
+      | org.camaraproject.geofencing-subscriptions.v0.area-entered |
+      | org.camaraproject.geofencing-subscriptions.v0.area-left    |
 
-  @geofencing_subscriptions_02_Create_geofencing_subscription_for_a_device_async
-  Scenario:  Create geofencing subscription (async creation)
-    Given that subscriptions are created asynchronously
+  @geofencing_subscriptions_01.1_sync_creation_3legs
+  Scenario Outline: Create geofencing subscription (async creation) with 3-legged-token
+    # Some implementations may only support asynchronous subscription creation
+    Given that subscriptions are created synchronously
```

Comment:
The previous scenario, both for sync case could be consolidated, as in the artifact